pkg/helm/charts/tests/centralservices/testdata/helmtest/scanner-v4.test.yaml

```diff
@@ -229,5 +229,25 @@ tests:
   set:
     scannerV4.disable: false
   expect: |
-    .configmaps["scanner-v4-indexer-config"].data.["config.yaml"] | fromyaml | .indexer.repository_to_cpe_url | assertThat(contains("https://central.stackrox.svc/api/extensions/scannerdefinitions"))
-    .configmaps["scanner-v4-indexer-config"].data.["config.yaml"] | fromyaml | .indexer.name_to_repos_url | assertThat(contains("https://central.stackrox.svc/api/extensions/scannerdefinitions"))
+    .configmaps["scanner-v4-indexer-config"].data.["config.yaml"] | fromyaml | .indexer.repository_to_cpe_url | assertThat(. == "https://central.stackrox.svc/api/extensions/scannerdefinitions?file=repo2cpe")
+    .configmaps["scanner-v4-indexer-config"].data.["config.yaml"] | fromyaml | .indexer.name_to_repos_url | assertThat(. == "https://central.stackrox.svc/api/extensions/scannerdefinitions?file=name2cpe")
+
+- name: "indexer should allow ingress from central and matcher when deployed as a central-service"
+  set:
+    scannerV4.disable: false
+  expect: |
+    .networkpolicys["scanner-v4-indexer"].spec.ingress | assertThat(length == 3)
```

Comment:
how much work would it be to convert `networkpolicys` to `networkpolicies`?

Reply:
Don't know from the top of my head. Would need to check in helmtest.